paasta_tools/kubernetes_tools.py

```diff
@@ -964,6 +960,20 @@ def should_run_uwsgi_exporter_sidecar(
                     return True
         return False
 
+    def should_setup_piscina_prometheus_scraping(
+        self, system_paasta_config: SystemPaastaConfig,
+    ) -> bool:
+        if self.is_autoscaling_enabled():
+            autoscaling_params = self.get_autoscaling_params()
+            if autoscaling_params["metrics_provider"] == "piscina":
+                if autoscaling_params.get(
+                    "use_prometheus",
+                    DEFAULT_USE_PROMETHEUS_PISCINA
+                    or system_paasta_config.default_should_setup_piscina_prometheus_scraping(),
+                ):
+                    return True
+        return False
```

Comment:
imo, we can skip adding awareness of `use_prometheus` here - this was meant as a toggle while we were transitioning from sfx to prometheus for uwsgi-based autoscaling. 

since this is something completely new, we can get away with just doing something like
```suggestion
    def should_setup_piscina_prometheus_scraping(self) -> bool:
        if self.is_autoscaling_enabled():
            autoscaling_params = self.get_autoscaling_params()
        
        return False
```

and uh, eventually cleanup `use_prometheus` for uwsgi :p (I think the only place we'd want to leave use_prometheus is for moving average cpu utilization scaling, which we just need to get back to at some point and debug)

Reply:
We still need ` if autoscaling_params["metrics_provider"] == "piscina":` though right?

Reply:
uh, yea - I'm just bad at writing code in a textfield ;_;
```suggestion
    def should_setup_piscina_prometheus_scraping(
        self, system_paasta_config: SystemPaastaConfig,
    ) -> bool:
        if self.is_autoscaling_enabled():
            autoscaling_params = self.get_autoscaling_params()
            return autoscaling_params["metrics_provider"] == "piscina":
        return False
```
is what i meant